enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/actions/CreateContainerRepositoryCommand.java

```diff
@@ -34,19 +35,19 @@ public class CreateContainerRepositoryCommand extends CreateResourceCommand<Cont
     
     @Override
     ContainerRepositoryItem callCreate(OCIItemCreationDetails itemCreator) {
-        ArtifactsClient client = this.getProfile().newClient(ArtifactsClient.class);
+        ArtifactsClient client = OCIManager.getDefault().getActiveProfile(itemCreator.getCompartment()).newClient(ArtifactsClient.class);
         
         CreateContainerRepositoryRequest request = (CreateContainerRepositoryRequest) itemCreator.getRequest();
         CreateContainerRepositoryResponse response = client.createContainerRepository(request);
         ContainerRepository res = response.getContainerRepository();
 
-        String tenancyId = this.getProfile().getTenancy().isPresent() ? this.getProfile().getTenancy().get().getKey().getValue() : null;
+        String tenancyId = itemCreator.getCompartment().getRegionCode();
```

Comment:
Seems as a miscompleted change ? If the `regionCode` is the correct value, pls. rename the local variable, so it is obvious that it is not an OCID.